nix/scripts.nix

```diff
@@ -6,12 +6,12 @@
 
 }: rec {
   start-chainmain = pkgs.writeShellScriptBin "start-chainmain" ''
-    export PATH=${pkgs.pystarport}/bin:${chainmain}/bin:$PATH
+    export PATH=${pkgs.test-env}/bin:${chainmain}/bin:$PATH
     ${../scripts/start-chainmain} ${config.chainmain-config} ${config.dotenv} $@
   '';
   start-cronos = pkgs.writeShellScriptBin "start-cronos" ''
     # rely on environment to provide cronosd
-    export PATH=${pkgs.pystarport}/bin:$PATH
+    export PATH=${pkgs.test-env}/bin:$PATH
     ${../scripts/start-cronos} ${config.cronos-config} ${config.dotenv} $@
   '';
   start-geth = pkgs.writeShellScriptBin "start-geth" ''
```

Comment:
start-geth might also need `test-env`

Reply:
yeah, indeed, strange that it works before.